TODO: Build and run on dotnet core on Ubuntu

Currently unable to build and run on dotnet core on Ubuntu -- fails with error:
```
/home/runner/.dotnet/sdk/3.1.405/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3644: The reference assemblies for .NETFramework,Version=v4.8 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks [/home/runner/work/HelloOwin/HelloOwin/HelloOwinServer/HelloOwinServer.csproj]
```

--- a/.github/workflows/dotnetcore.yml
+++ b/.github/workflows/dotnetcore.yml
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         dotnet: [ '3.1.x', '6.0.x', '7.0.x' ]
-        os: [ 'windows-latest' ]
+        os: [ 'windows-latest', 'macos-latest', 'ubuntu-latest' ]
     name: DotNet ${{ matrix.dotnet }} ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     steps:
@@ -24,4 +24,3 @@ jobs:
         run: dotnet build --configuration Release
       - name: Test with dotnet
         run: dotnet test --configuration Release
-

